src/AttestationData.ts

```diff
@@ -0,0 +1,408 @@
+type TContextField = string | {type: string; data: string}
+
+///////////////////////////////////////////////////
+// Base attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAtt {
+  '@context': TContextField | Array<TContextField>
+  // Date/time attestation was performed
+  date?: string
```

Comment:
Should this be `TDate`? Or maybe a comment for what format the date will be in

Reply:
Yes, TDate was introduced halfway through the PR, fixed